Docs/RunTestLocally.md

```diff
@@ -0,0 +1,68 @@
+
+- CheepRepositoryUnitTests
+<br> This class targets our AuthorRepository. It performs unit tests on liking a cheep. e.g. liking increases a cheeps total likes.
+
+- InMemoryDatabaseTests
+<br> This class tests if the in memory database is created correctly which is crucial for the other classes since they all rely on it.
+
+- RestrictedCheepsUnitTests
+<br> This class targets the Cheepvalidator. It performs unit test on if a cheep has the correct information we want from it. e.g. A cheep not being empty or over 160 chars, and has a valid author.
+
+
+## Razor.Tests
+To run the tests you need to setup and download docker. A complete guide for downloading and setting up docker correctly with our application can be found in our [README.md](..\README.md#docker-setup)
+After following the guide cd into the Chirp.Razor.tests folder and run the following command
+```bash
+dotnet test
+```
+### what is tested
+The razor tests consist of one class, **IntegrationTest.cs**. The class creates a local instance of our web application using the [WebApplicationFactory class](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.testing.webapplicationfactory-1?view=aspnetcore-8.0). With this we can test that our applications ui functions as we expect before we deploy it to azure. The test include, testing 32 cheeps per page, ordering of cheeps by date and the functionality of dynamic buttons.
+
+## Playwright.tests
+To run the test first download playwright with the following command
+
+  ```bash
+  pwsh bin/Debug/net7.0/playwright.ps1 install
+  ```
+This install various browsers and tools to run UI tests. The browser we use is chromium based.
+<br>
+if you run in to issues with the version of .net replace net7.0 in the command with the correct version
+<br>
+if you don't have powerShell installed follow these instructions
+[Install PowerShell](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.4)
+
+Extra logic for authorization hasn't been implemented so the developer has to manually enter their github username into the variable at line 18 in the **PlaywrightTests.cs** file. Further explanation is also found there. After completing these steps you can run the test with: 
+```bash 
+dotnet test 
+  ```
+When you run the test a chromium based browser will open and the first step tries to login. Here the automation stops and the user has to login through github themselves. **No passwords are saved!** After this step is completed playwright will do the rest itself.
+
+### what is tested
+The playwright test differs from the razor test in that it more mimics user behavior on our live website compared to the razor test which test locally. The test navigates through different pages and interacts with the website's functionality confirming that what it interacts with is as expected in the test. 
```

Comment:
A quantifier is used incorrectly here "that it more mimics user" it should be "that, it mimics user"

compared to the razor test which test**s** locally